api/nodes/serializers.py

```diff
@@ -63,10 +40,12 @@ class NodeSerializer(JSONAPISerializer):
                                                             'public and private nodes. Administrators on a parent '
                                                             'node have implicit read permissions for all child nodes',
                               )
-    # TODO: finish me
 
-    class Meta:
-        type_ = 'nodes'
+    registration = ser.BooleanField(source='is_registration', read_only=True)
```

Comment:
We had a properties dictionary to collect all of that, but that went for reasons. That being said, we have a schema in the new API, so we don't need to rely on naming conventions for that sort of thing.